mailer/CHANGELOG.md

```diff
@@ -1,229 +0,0 @@
-<a name="1.81.0"></a>
```

Comment:
In an ideal world, would we merge this with the other `CHANGELOG.md`? Or keep it separately somewhere? Obviously it's no big deal, but if it was worth generating I wonder if it's worth keeping is all.